elodie/tests/filesystem_test.py

```diff
@@ -506,8 +507,10 @@ def test_process_file_plain():
     shutil.rmtree(folder)
     shutil.rmtree(os.path.dirname(os.path.dirname(destination)))
 
-    assert origin_checksum is not None, origin_checksum
-    assert origin_checksum == destination_checksum, destination_checksum
+    assert origin_checksum_preprocess is not None
+    assert origin_checksum is not None
+    assert destination_checksum is not None
+    assert origin_checksum_preprocess  == origin_checksum
```

Comment:
There's an extra space before the `==` in this and all of the duplicated lines below.